src/components/widget/Widget.scss

```diff
@@ -21,23 +21,36 @@
     font-weight: bold;
 }
 
-.inner.widget-triangle::after {
-    /* The downward triangle below widget. */
-
-    //KG todo replace the annotator's triangle with our own
-    background-image: url(../../../node_modules/annotator/img/annotator-glyph-sprite.png?embed);
-
-    background-repeat: no-repeat;
-
-    content: "";
-    display: block;
-    width: 18px;
-    height: 10px;
-    background-position: 0 0;
-    position: absolute;
-    bottom: -10px;
-    left: 8px;
 
+/*
+* The downward triangle below widget.
+* It is made of two triangles, smaller and bigger. The bigger triangle forms the border around the smaller one.
+*/
+.inner.widget-triangle {
+    &::before, &::after {
+        content: "";
+        display: block;
+        position: absolute;
+        /* The points are: centered top, left bottom, right bottom */
+        clip-path: polygon(50% 100%, 0 0, 100% 0);
```

Comment:
is it safe to use? Support [isn't very rock solid](https://caniuse.com/#search=clip-path)

Reply:
I found a much better solution in `semantic-ui` tooltips and adopted it right away :) (rotating a square instead of clipping it)